tests/datatypes/test_bytes.py

```diff
@@ -105,23 +105,23 @@ class BinaryBytesOperationTests(BinaryOperationTestCase, TranspileTestCase):
     data_type = 'bytes'
 
     not_implemented = [
-        'test_add_bool',
+        # 'test_add_bool',
```

Comment:
I would just delete these lines rather than comment them out. We won't need to add them back in again hopefully :)